File: app/src/main/java/com/jimzrt/umsmounter/Model/Release.java
package com.jimzrt.umsmounter.Model;

import com.jimzrt.umsmounter.Utils.Helper;

public class Release {
    public String url;
    public String version;
    public String size;

    public String toString() {
        return url.substring(url.lastIndexOf('/') + 1, url.length()) + "\nversion: " + version + "\nsize: " + Helper.humanReadableByteCount(Long.parseLong(size));
    }
}

File: app/src/main/java/com/jimzrt/umsmounter/model/Release.kt
package com.jimzrt.umsmounter.model

import com.jimzrt.umsmounter.utils.Helper.humanReadableByteCount

class Release {
    var url: String? = null
    private val version: String? = null
    private val size: String? = null
    override fun toString(): String {
        return """
            ${url!!.substring(url!!.lastIndexOf('/') + 1)}
            version: $version
            size: ${humanReadableByteCount(size!!.toLong())}
            """.trimIndent()
    }
}